static/script.js

```diff
@@ -0,0 +1,10 @@
+function handleChange(e) {
+  if (e.matches) {
```

Comment:
So this event has a `matches` key, what exactly does that contain? More just out of curiosity.

Reply:
`matches` is a boolean that says if the media query matches

Reply:
oh wow, super cool. Thanks!